app/services/foreman_ansible/inventory_creator.rb

```diff
@@ -8,11 +8,42 @@ def initialize(hosts)
       @hosts = hosts
     end
 
-    def tempfile
-      tempfile = Tempfile.new("foreman-#{SecureRandom.uuid}-inventory")
-      tempfile.write(hosts.map(&:fqdn).join('\n'))
-      tempfile.close
-      tempfile
+    def json
```

Comment:
Maybe `hash` or `to_hash` would be better. It seems counter-intuitive to have to call `.json.to_json` to actually get the json